<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelInventory">
          <property name="text">
           <string>Количество затрат на пополнение запасов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtInventory">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelMaterials">
          <property name="text">
           <string>Количество затрат на хранение сырья на складах</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtMaterials">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelT">
          <property name="text">
           <string>Количество месяцев</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtT">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelDelta">
          <property name="text">
           <string>Пополнение по</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtDelta">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labelM">
          <property name="text">
           <string>Размер склада</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtM">
          <property name="text">
           <string>300</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelBeginMaterials">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Начальный запас</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtBeginMaterials">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBeginMaterials">
          <property name="text">
           <string>есть</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnWrite">
        <property name="text">
         <string>Заполнить начальные данные</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
